src/display.c

```diff
@@ -249,6 +249,22 @@ map_trap(trap, show)
     if (show) show_glyph(x, y, glyph);
 }
 
+int
+obj_to_glyph(obj)
+    struct obj *obj;
+{
+    int otyp = Hallucination ? random_object() : obj->otyp;
+
+    if (otyp == CORPSE) {
+	int corpsenm = Hallucination ? random_monster() : obj->corpsenm;
+	return corpsenm + GLYPH_MON_OFF;
```

Comment:
Shouldn't the constant be `GLYPH_BODY_OFF`?

Reply:
You're right, this should be corrected in the new commit :+1: